CHANGELOG

```diff
@@ -12,8 +12,6 @@ Breaking Changes
 - MissingTrieNode is no longer a KeyError, paving the way for eventually raising a KeyError instead
   of returning b'' when a key is not present in the trie
   https://github.com/ethereum/py-trie/pull/98
-- If traversing, instead of getting a key value, then MissingTrieNode.requested_key might be None
-  https://github.com/ethereum/py-trie/pull/98
```

Comment:
I decided this was too messy, and that a different exception with known values makes more sense (`MissingTraversalNode`)